rakelib/test.rake

```diff
@@ -4,7 +4,8 @@ namespace "test" do
     LogStash::Environment.bundler_setup!({:without => []})
     require "rspec/core/runner"
     require "rspec"
-    RSpec::Core::Runner.run([*args])
+    exit_code = RSpec::Core::Runner.run([*args])
```

Comment:
from what I've seen, using fail makes the rspec output itself fail, which gives an incorrect output. check this gist where I compared master, using fail, and using exit: https://gist.github.com/jsvd/718efda05ad1b5408b1b